pipeline_kafka.c

```diff
@@ -986,44 +1008,28 @@ kafka_consume_main(Datum arg)
 				rd_kafka_message_destroy(messages[i]);
 				messages[i] = NULL;
 			}
+
+			/* Flush if we've buffered enough messages or space used by messages has exceeded 8mb */
+			if (messages_buffered >= consumer.batch_size || buf->len > 8388608)
```

Comment:
In that case maybe we should also increase the hard coded copy max bytes, which right now is 64k.